api/Response.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "api": {
+    "Response": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Response",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
+          },
+          "chrome": {
+            "version_added": "42"
+          },
+          "chrome_android": {
+            "version_added": "42"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "39"
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "29"
+          },
+          "opera_android": {
+            "version_added": "29"
+          },
+          "safari": {
+            "version_added": "10.1"
+          },
+          "safari_ios": {
+            "version_added": "10.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "body_formdata": {
```

Comment:
Let's remove this. It is stored here https://github.com/mdn/browser-compat-data/blob/master/api/Body.json#L380

Reply:
👍